VideoLocker/res/layout/edx_spinner_dropdown_item.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CheckedTextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@android:id/text1"
```

Comment:
You're right, I made an assumption. I started with Android's own simple_spinner_item layout which had this ID so I assumed it used it.